### PR TITLE
import cluster workflow with mocked data

### PR DESCRIPTION
**Pull request by kamleshverma1**
- not able to resolve one test case issue in host-list-test.js (line no - 70) after updation

Signed-off-by: kamleshKumarVerma <kaverma@redhat.com>

### CODE REVIEW

**Comment by kamleshverma1** [comment]
@gnehapk please review it . and - i am not able to resolve one test case issue in host-list-test.js (line no - 70) after host-controller updation

**Comment by kamleshverma1** [comment]
@cloudbehl Please review it

File: src/modules/cluster/import-cluster/import-cluster.js [hunk 1]
```diff
@@ -6,32 +6,88 @@
...
+        var vm = this, hostList = [], hostDetail = {};
+        
+        vm.step = 1;
+        vm.detectedClusters = [];
+        vm.init = init;
+        vm.importCluster = importCluster;
+        vm.importCancel = importCancel;
+        vm.viewTaskProgress = viewTaskProgress;
+        vm.selectCluster = selectCluster;
+        vm.init();
```
Comment (gnehapk) [inline src/modules/cluster/import-cluster/import-cluster.js]: @kamleshverma1 Put only those functions on controller instance which needs to be accessed from html. Please remove init from vm instance.

File: src/commons/services/utils.js [hunk 2]
```diff
@@ -96,6 +96,24 @@
             });
         };
 
+        vm.importCluster = function(uri, data) {
+            var url, actionRequest, request;
+
+            url = config.baseUrl + uri;
+
+            actionRequest = {
```
Comment (cloudbehl) [inline src/commons/services/utils.js]: Utils.js will have file contains get and post request many times, So can we create two different functions. One for creating a Get request and another for Post request. So we done have to write things again and again.

Comment (kamleshverma1) [inline src/commons/services/utils.js]: @cloudbehl Ack , valid point , will update in next PR . actually this PR got merged .

Comment (cloudbehl) [inline src/commons/services/utils.js]: @kamleshverma1 No problem.

**Pull request merged**